src/Features.js

```diff
@@ -22,7 +22,7 @@ const Features = () => (
     <div className='feature container'>
       <div className='right visual'><i className='ion-ios-people' /></div>
       <div className='left text'>
-        <h2><FormattedMessage id='features.user_first' defaultMessage='Putting the user first' /></h2>
+        <h2><FormattedMessage id='features.user_first' defaultMessage='Putting people first' /></h2>
```

Comment:
You need to also change the string in src/locales/en.json to actually see the change - this line only controls the default value for new locales